### DISCUSSION TITLE
Você tá ficando velho #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#523] [Curiosidades]
Post de referência: https://www.vish.blog.br/artigos/voce-ta-ficando-velho-1/

**Comment by brunoaugusto** [comment]
Eu depois de ler o artigo



![null](<https://i.postimg.cc/qqKJrypy/back-pain-old-man.png>)

**Comment by one-eyed-willy** [comment]
Quando o careca começou a dançar no vídeo do Centro Espacial eu não aguentei XD

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/artigos/voce-ta-ficando-velho-1/#comment_1087601230>):

> Quando o careca começou a dançar no vídeo do Centro Espacial eu não aguentei XD

Ele olhando feio pro ator eu já pensei "o careca vai dançar também, quer ver" ^^

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/artigos/voce-ta-ficando-velho-1/#comment_1087603983>):

> Ele olhando feio pro ator eu já pensei "o careca vai dançar também, quer ver" ^^

Pena eu só ter visto esse vídeo 10 anos depois que foi postado porque ficou incrível!

**Comment by vish-blog** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/artigos/voce-ta-ficando-velho-1/#comment_1087601230>):

> Quando o careca começou a dançar no vídeo do Centro Espacial eu não aguentei XD

Esse foi apenas um dos motivos pelo qual esse vídeo entrou pro post dentre todos os que eu assisti — e não foram poucos
